catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -106,32 +102,31 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
     try (Connection connection = getConnection(databaseName)) {
-      String comment;
-      Map<String, String> properties;
-      try (ResultSet table = getTable(connection, tableName)) {
-        if (!table.next()) {
-          throw new NoSuchTableException("Table " + tableName + " does not exist.");
-        }
-        comment = table.getString("REMARKS");
-        properties = extractPropertiesFromResultSet(table);
+      // 1.Get table information
+      ResultSet table = getTable(connection, databaseName, tableName);
+      if (!table.next() || !tableName.equals(table.getString("TABLE_NAME"))) {
+        throw new NoSuchTableException(
+            String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
-      JdbcColumn[] jdbcColumns;
-      try (ResultSet column = getColumns(connection, tableName)) {
-        List<JdbcColumn> result = new ArrayList<>();
-        while (column.next()) {
-          result.add(extractJdbcColumnFromResultSet(column));
-        }
-        jdbcColumns = result.toArray(new JdbcColumn[0]);
+
+      // 2.Get column information
+      List<JdbcColumn> jdbcColumns = new ArrayList<>();
+      ResultSet columns = getColumns(connection, databaseName, tableName);
+      while (columns.next()) {
+        JdbcColumn.Builder columnBuilder = createJdbcColumnBuilder(columns);
+        extractJdbcColumnFromResultSet(columns, columnBuilder);
```

Comment:
`extractJdbcColumnFromResultSet` `extractJdbcTableFromResultSet` `extractJdbcTable` are too general names. I couldn't get much information from the name. may be we could use `getAutoIncreamentInfo()` `getIndexInfo()` to  get responding context.  `createJdbcXXBuilder` suggest to rename to `getBasicJdbcxxInfo`. the code maybe like
```java
        JdbcColumn.Builder columnBuilder = getBasicJdbcColumnInfo(columns);
        Bool autoIncrement = getAutoIncreamentInfo(columns);
        collumnBuilder.withAutoIncrement(autoIncrement)
```